python/paddle/fluid/tests/unittests/dygraph_to_static/transformer_dygraph_model.py

```diff
@@ -486,3 +488,170 @@ def forward(self, src_word, src_pos, src_slf_attn_bias, trg_word, trg_pos,
         predict = self.decoder(trg_word, trg_pos, trg_slf_attn_bias,
                                trg_src_attn_bias, enc_output)
         return predict
+
+    @dygraph_to_static_func
+    def beam_search(
+            self,
+            src_word,
+            src_pos,
+            src_slf_attn_bias,
+            trg_word,
+            trg_src_attn_bias,
+            bos_id=0,
+            eos_id=1,
+            beam_size=4,
+            max_len=256, ):
```

Comment:
please remove the comma "max_len=256," here

Reply:
Thanks. Done.